app/src/main/java/com/infomaniak/drive/utils/Extensions.kt

```diff
@@ -803,6 +796,16 @@ fun Context.shareText(text: String) {
     ContextCompat.startActivity(this, Intent.createChooser(intent, null), null)
 }
 
+fun Context.getTintedDrawable(drawableId: Int, colorString: String): Drawable? {
+    return getTintedDrawable(drawableId, colorString.toColorInt())
+}
+
+fun Context.getTintedDrawable(drawableId: Int, colorInt: Int): Drawable? {
+    return ContextCompat.getDrawable(this, drawableId)
+        ?.mutate() // Mutate the drawable, so it won't change the tint when we use the same resource elsewhere.
```

Comment:
Now it's ok everything works fine with this?

Reply:
Yes, I wasn't able to reproduce the issue after adding this `mutate()`.